C/plugins/storage/postgres/plugin.cpp

```diff
@@ -39,7 +39,7 @@ static PLUGIN_INFORMATION info = {
 	"1.0.0",                  // Version
 	SP_COMMON|SP_READINGS,    // Flags
 	PLUGIN_TYPE_STORAGE,      // Type
-	"1.0.0"                   // Interface version
+	"1.1.0"                   // Interface version
```

Comment:
These 3 new functions in storage plugin should be called only if the interface version is >= "1.1.0". I didn't find that check in this PR.

Reply:
There is no check for that right now